src/vllm_tgis_adapter/__main__.py

```diff
@@ -3,17 +3,19 @@
 import asyncio
 import contextlib
 import os
+import socket
 import traceback
 from concurrent.futures import FIRST_COMPLETED
-from typing import TYPE_CHECKING
+from typing import TYPE_CHECKING, Tuple
```

Comment:
Tuple should be unused now, but the linter should catch this

Reply:
went back and ran the linter